src/shared/components/palette/collection/write-collection-field.html

```diff
@@ -19,9 +22,10 @@ <h2 class="heading-h2 error-spacing">
             <label [for]="buildIdPrefix(i) + i"><h3 class="heading-h3">{{itemLabel(i)}}</h3></label>
           </div>
           <div class="float-right">
-            <button class="button button-secondary" type="button" (click)="openModal(i)"
-                    [disabled]="isNotAuthorisedToDelete(i)"
+              <ng-container *ngIf="!isNotAuthorisedToDelete(i)">
```

Comment:
double negation - one from "!" and another from the function name
makes code less smooth to read